tests/unit/httpd_test.cc

```diff
@@ -1016,6 +1016,27 @@ static future<> test_basic_content(bool streamed, bool chunked_reply) {
                 BOOST_REQUIRE_EQUAL(body, sstring("12345 78901\t34521345"));
             }
 
+            {
+                fmt::print("Request with content-length body\n");
+                auto req = http::request::make("GET", "test", "/test");
+                req.write_body("txt", 12, [] (output_stream<char>&& out) {
+                    return seastar::async([out = std::move(out)] () mutable {
+                        out.write(sstring("1234567890")).get();
+                        out.write(sstring("ABCDEFGHIJ")).get();
```

Comment:
Please add a test that writes more than 32000 in one chunk, to exercise that trim_to_size.

(IIRC it was udp related)